src/clr-angular/modal/modal.html

```diff
@@ -12,7 +12,7 @@
          [class.modal-sm]="size == 'sm'"
          [class.modal-lg]="size == 'lg'"
          [class.modal-xl]="size == 'xl'"
-         role="dialog" aria-hidden="true">
+         role="dialog">
```

Comment:
Contrary to what @Shijir said, I'd feel safer if this became `aria-hidden="!_open"` instead of removing it entirely.
The content will still be readable by screenreaders when the modal is open, but if for some reason our `*ngIf`s around change, the modal will have a proper `aria-hidden="true"` when closed.